sales_team_security/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Security rules for sales teams
+==============================
+
+This module sets different permissions levels for accessing sales records
+based on the sales team: customers, sales orders and sales teams.
+
```

Comment:
Does it means that a user only will see customers, sale orders and sale teams in which he is assigned?